### PR TITLE
adjust performance numbers based on new benchmark results

### PR DESCRIPTION
**Pull request by hisImminence**
## Description

This PR updates the performance numbers based on the new results of the benchmarks (see [slack message](https://camunda.slack.com/archives/C07J9URA0B1/p1729846009457529) from @rodrigo-lourenco-lopes ). 

These values are just valid for 8.6+.

* Assigning @rodrigo-lourenco-lopes to double check the numbers.
* Assigning @mesellings as you had been responsible for the docs in the first place as a FYI and to make sure I dont change anything unexpected :D 

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [x] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory (--> just valid for 8.6).
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/best-practices/architecture/sizing-your-environment.md [hunk 1]
```diff
@@ -151,9 +151,9 @@ Contact your Customer Success Manager to increase the cluster size beyond the ma
...
-| Max Throughput **Tasks/second** **\***                                              |                                 50 |                                 100 |                              150 |                              200 |
-| Max Throughput **Process Instances/day** **\*\***                                   |                                3 M |                                 6 M |                              9 M |                             12 M |
+| Max Throughput **Tasks/day** **\***                                                 |                                9 M |                                18 M |                             27 M |                             36 M |
+| Max Throughput **Tasks/second** **\***                                              |                                100 |                                 200 |                              300 |                              400 |
+| Max Throughput **Process Instances/day** **\*\***                                   |                              432 k |                               864 k |                          1.296 k |                          1.728 k |
```
Comment (mesellings) [inline docs/components/best-practices/architecture/sizing-your-environment.md:156]:
```suggestion
| Max Throughput **Process Instances/day** **\*\***                                   |                              432 k |                               864 k |                          1,296 k |                          1,728 k |
```
I think the US uses commas as a numerical separator?

File: versioned_docs/version-8.6/components/best-practices/architecture/sizing-your-environment.md [hunk 2]
```diff
@@ -151,9 +151,9 @@ Contact your Customer Success Manager to increase the cluster size beyond the ma
...
-| Max Throughput **Tasks/second** **\***                                              |                                 50 |                                 100 |                              150 |                              200 |
-| Max Throughput **Process Instances/day** **\*\***                                   |                                3 M |                                 6 M |                              9 M |                             12 M |
+| Max Throughput **Tasks/day** **\***                                                 |                                9 M |                                18 M |                             27 M |                             36 M |
+| Max Throughput **Tasks/second** **\***                                              |                                100 |                                 200 |                              300 |                              400 |
+| Max Throughput **Process Instances/day** **\*\***                                   |                              432 k |                               864 k |                          1.296 k |                          1.728 k |
```
Comment (mesellings) [inline versioned_docs/version-8.6/components/best-practices/architecture/sizing-your-environment.md:156]:
```suggestion
| Max Throughput **Process Instances/day** **\*\***                                   |                              432 k |                               864 k |                          1,296 k |                          1,728 k |
```
See above suggestion

**Review by mesellings — APPROVED** [review]
I think the comma numeric separator should be changed possibly, other than that it looks good from a writing standpoint 👍

**Review by rodrigo-lourenco-lopes — APPROVED** [review]
Thanks for the changes! 🚀

File: docs/components/best-practices/architecture/sizing-your-environment.md (hunk 1 above)
Comment (akeller) [reply docs/components/best-practices/architecture/sizing-your-environment.md:156]: We do, but in this case, we would use a decimal. When digit grouping, we use a comma.

Comment (mesellings) [reply docs/components/best-practices/architecture/sizing-your-environment.md:156]: Thanks for the clarification @akeller, resolving this comment without an edit needed 👍

**Review by mesellings — APPROVED** [review]
Lgtm 👍 🚀

**Comment by mesellings** [comment]
@akeller I'll do a docs release to get this out 👍

**Pull request merged**